### PR TITLE
added update arg and logic

### PR DESCRIPTION
**Pull request by Johnmancini30**
Adding an `try-convert -u|--update` command that will update the tool to the latest version.

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/try-convert?pullRequest=401) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/try-convert?pullRequest=401) before we can accept your contribution.<br/><br/>:x: Johnmancini30 [sign now](https://cla.dotnetfoundation.org/dotnet/try-convert?pullRequest=401)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/try-convert?pullRequest=401) it.</sub>

File: src/try-convert/UpdateTryConvert.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+{
+    public static class UpdateTryConvert
+    {
+        public static void Update()
+        {
+            using (var process = new Process())
+            {
+                process.StartInfo.UseShellExecute = true;
+                process.StartInfo.FileName = "dotnet";
+                process.StartInfo.Arguments = "tool update --global try-convert";
```
Comment (jmarolf) [inline src/try-convert/UpdateTryConvert.cs:13]: @Johnmancini30 what is the tool is not installed as a global tool?

Comment (Johnmancini30) [reply src/try-convert/UpdateTryConvert.cs]: Okay, responded

**Review by cartermp — APPROVED** [review]
This looks good to me!

**Comment by Johnmancini30** [comment]
Thanks! I cannot merge this FYI

**Comment by cartermp** [comment]
Thanks! Will spin up a new release

**Pull request merged**